spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/repository/JdbcDataflowTaskExecutionDaoTests.java

```diff
@@ -56,18 +53,14 @@ public class JdbcDataflowTaskExecutionDaoTests {
 
 	@Autowired
 	private TaskRepository taskRepository;
-	@Autowired
-	private AggregateExecutionSupport aggregateExecutionSupport;
-	@Autowired
-	private TaskDefinitionReader taskDefinitionReader;
 
 	@Test
 	@DirtiesContext
 	public void testGetTaskExecutionIdsByTaskName() {
 		String taskName = UUID.randomUUID().toString();
-		List<AggregateTaskExecution> taskExecutions = createSampleTaskExecutions(taskName, 4);
-		for (AggregateTaskExecution taskExecution : taskExecutions) {
-			taskRepository.createTaskExecution(taskExecution.toTaskExecution());
+		List<TaskExecution> taskExecutions = createSampleTaskExecutions(taskName, 4);
+		for (TaskExecution taskExecution : taskExecutions) {
+			taskRepository.createTaskExecution(taskExecution);
 		}
 		assertThat(dataflowTaskExecutionDao).isNotNull();
 		Set<Long> taskExecutionIds = dataflowTaskExecutionDao.getTaskExecutionIdsByTaskName(taskName);
```

Comment:
While I have focused on this test for some reason.... this can be consolidated a bit as well. 

```
assertThat(dataflowTaskExecutionDao.getTaskExecutionIdsByTaskName(taskName)).hasSize(4);
```